vector-test/src/test/scala/spec/geotrellis/vector/voronoi/DelaunaySpec.scala

```diff
@@ -102,13 +111,42 @@ class DelaunaySpec extends FunSpec with Matchers {
       // NOTE: In the event of failure, the following line will draw the triangulation
       // to delaunay.png in the working directory, indicating which triangle did not
       // exhibit the Delaunay property
-      //rasterizeDT(dt)
+      // rasterizeDT(dt)
 
       (dt.triangles.forall{ case ((ai,bi,ci),_) =>
         val otherPts = (0 until numpts).filter{ i: Int => i != ai && i != bi && i != ci }
         otherPts.forall{ i => ! localInCircle((ai,bi,ci),i) }
       }) should be (true)
     }
+
+    it("should work on a real dataset with many collinear points") {
+      val allcities = ResourceReader.asString("populous_cities.txt")
```

Comment:
We don't normally read test data out of resources packed in the JAR - is there an advantage you see of doing it this way, other then adding to test files similar to other tests? We would put it here https://github.com/geotrellis/geotrellis/tree/master/vector-test/data